CMakeLists.txt

```diff
@@ -372,7 +372,6 @@ set(NEODLR_SRC ${DLR_SRC})
 list(REMOVE_ITEM NEODLR_SRC 
     ${TVM_SRC}/src/runtime/dso_library.cc
     ${TVM_SRC}/src/runtime/cpu_device_api.cc
-    ${TVM_SRC}/src/runtime/metadata_module.cc
```

Comment:
?

Reply:
essentially we trying to remove lines 107-110. this got added by mistake while cherrypicking but it is redundant. hence remove it in the patch.